android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -272,6 +274,8 @@ public void showMapButtons(boolean show)
       UiUtils.show(mFrame);
     else
       UiUtils.hide(mFrame);
+
+    resumeZoomButtons();
```

Comment:
As said in another comment, you can use `updateButtonsVisibility` directly. To improve performance it is also better to call this function only when `show` is true: we don't need to loop over all the buttons to detect if they should show or not if we want to hide them all